include/RED4ext/Dump/Reflection-inl.hpp

```diff
@@ -41,10 +41,20 @@ RED4EXT_INLINE void Dump(std::filesystem::path aOutPath, std::filesystem::path a
     {
         size_t i = 0;
 
-        // Special case for AI
-        if (aInput.size() >= 2 && aInput[0] == 'A' && aInput[1] == 'I')
+        constexpr std::string_view exceptions[] = {"AI", "GpuWrapApi", "GpuWrapApiVertexPacking"};
+
+        for (const auto& exception : exceptions)
+        {
+            if (aInput.size() > exception.size() && aInput.starts_with(exception))
+            {
+                i = exception.size();
+            }
+        }
+
+        // Special case for enums under `GpuWrapApi` i.e. `eTextureType`
+        if ((aInput.starts_with("GpuWrapApi") || aInput.starts_with("GpuWrapApiVertexPacking")) && aInput[i] == 'e')
         {
-            i = 2;
+            return aInput.substr(0, i);
```

Comment:
If you need to do this, then it might be worth to have the `constexpr` variable a `std::pair` with a boolean, if true return else do `i = size`.